prow/cmd/peribolos/main.go

```diff
@@ -440,6 +440,11 @@ func configureOrgMembers(opt options, client orgClient, orgName string, orgConfi
 		om, err := client.UpdateOrgMembership(orgName, user, super)
 		if err != nil {
 			logrus.WithError(err).Warnf("UpdateOrgMembership(%s, %s, %t) failed", orgName, user, super)
+			if github.IsNotFound(err) {
```

Comment:
Any chance we can eventually exit non zero in this situation?

Reply:
Not the way this is written now, no -- we don't append the error if it's `nil` so we won't error out on this